raiden/exceptions.py

```diff
@@ -161,6 +161,18 @@ class InvalidTokenAddress(RaidenError):
     """ Raised if the token address is invalid """
 
 
+class InvalidTokenNetworkDepositLimit(RaidenError):
+    """ Raised when an invalid token network deposit
+    limit is passed to the token network registry proxy.
+    """
```

Comment:
This can happen only when the API didn't check enough conditions.  This should crash the node with `BrokenPreconditionError`.